nifi-docs/src/main/asciidoc/nifi-in-depth.adoc

```diff
@@ -34,26 +34,28 @@ There are three repositories that are utilized by NiFi. Each exists within the O
 - The Content Repository holds the content for current and past FlowFiles.
 - The Provenance Repository holds the history of FlowFiles.
 
-image::NiFiArchitecture.png["NiFi Architecture Diagram"]
+image::zero-master-node.png["NiFi Architecture Diagram"]
```

Comment:
Ah one thing I almost missed, can I remove the NiFiArchitecture.png image? I don't see it referenced anywhere else. I can just amend it to your commit.